gradle.properties

```diff
@@ -3,9 +3,9 @@ org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
 	# check these on https://fabricmc.net/use
-	minecraft_version=1.16.2-pre1
-	yarn_mappings=1.16.2-pre1+build.9
-	loader_version=0.9.0+build.204
+	minecraft_version=1.18-pre1
+	yarn_mappings=1.18-pre1+build.3
+	loader_version=0.12.5
 
 # Mod Properties
 	mod_version = 3.0.1+j1.2.0
```

Comment:
Let's get this changed to 3.0.1+j1.2.1

Reply:
I used 4.0.0 after merging (the MC version update is a breaking change), but forgot to change the Jankson version here